### ISSUE TITLE
User notifications

### ISSUE THREAD

**Issue by bajiat** [apinf/platform#2248]
# User story
> As an API owner,
I want to be able to see notifications concerning the APIs I manage
so that I can take necessary actions.

> As an API consumer
I want to be able to see notifications concerning the APIs I follow or use
so that I can make changes to my applications or I'm aware of for example about my spending (monetization).

# Goal
APInf is able to show a history of user related notifications on the UI, which can be discarded (marked as read) by the user.

**Comment by bajiat** [comment]
@Nazarah Definitely. This issue is related to the frontpage future enhancements, which include following API.

**Comment by Nazarah** [comment]
There is a really [good article](https://uxplanet.org/how-to-design-notifications-for-better-ux-6fb0711be54d) in medium to know about Notifications types a the UX we should consider while designing notifications.
